src/Connection.js

```diff
@@ -439,6 +439,22 @@ export function DatasourceModal({ selectedDataSource, connectionData }) {
     // setOtherSelectedMethod(null);
   }
 
+  const createTags = (methodVersions) => {
```

Comment:
This can be solved in a more browser-native way by removing the CSS for the div surrounding the tags.